cmd/krew/cmd/info.go

```diff
@@ -25,38 +25,41 @@ import (
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 
-	"sigs.k8s.io/krew/internal/info"
+	"sigs.k8s.io/krew/internal/index/indexscanner"
 	"sigs.k8s.io/krew/internal/installation"
+	"sigs.k8s.io/krew/internal/pathutil"
 	"sigs.k8s.io/krew/pkg/index"
 )
 
 // infoCmd represents the info command
 var infoCmd = &cobra.Command{
 	Use:   "info",
-	Short: "Show information about a kubectl plugin",
-	Long: `Show information about a kubectl plugin.
+	Short: "Show information about an available plugin",
+	Long: `Show detailed information about an available plugin.
 
-This command can be used to print information such as its download URL, last
-available version, platform availability and the caveats.
+Examples:
+  kubectl krew info PLUGIN
```

Comment:
We should say that this looks in the default index. Or is there a better place to explain the index resolution logic?

Reply:
IMO not needed since it’s implied from “search” and “list” outputs. 

Similarly brew commands use a x/taps/y syntax but they just call it FORMULA. They don’t really make it too obvious to everyone that taps concept exists as majority doesn’t need to know.

Reply:
Ok, makes sense.